src/Makefile.am

```diff
@@ -150,7 +179,7 @@ pveclib_la_INCLUDES = \
 	$(pveclibinclude_HEADERS)
 
 pveclib_test_la_INCLUDES = \
-       $(pveclibinclude_HEADERS)
+	$(pveclibinclude_HEADERS)
```

Comment:
not sure what this change is... changing a tab to spaces, or vice-versa? it could all fit on one line.

Reply:
ok all on one line